### PR TITLE
chore: improve documentation for the monitoring add-on

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]
Thank you

File: 01.Get-started/05.Monitor/docs.md [hunk 1]
```diff
@@ -47,14 +47,14 @@ shows how to detect that a USB-connected peripheral gets disconnected.
...
-mender-monitorctl enable log usb_disconnect
+sudo mender-monitorctl enable log usb_disconnect
 ```
 
- Now remove a USB device from the device (you can insert it first, e.g. a
- thumbdrive or mouse, if you don't have any USB devices inserted). The Alert now
- triggers in the Mender UI once you remove the USB device:
+Now remove a USB device from the device (you can insert it first, e.g. a
+thumbdrive or mouse, if you don't have any USB devices inserted). The Alert now
+triggers in the Mender UI, in the device details, once you remove the USB device:
```
Comment (merlin-northern) [inline 01.Get-started/05.Monitor/docs.md:55]: suggested change: "The Alert now...device:" -> "_Once you remove the USB device, the log subsystem triggers an alert which you can inspect  in the device details in the Mender UI:_"

File: 01.Get-started/05.Monitor/docs.md [hunk 2]
```diff
@@ -105,6 +105,12 @@ And an alert shows up in the UI.
 
 ![Disk usage alarm triggered](diskusage-alarm.png)
 
+If you remove the file, the alert hides in the UI:
```
Comment (merlin-northern) [inline 01.Get-started/05.Monitor/docs.md:108]: I would rephrase "the alert hides" somehow it does not seem a good fit, perhaps "the alert is cancelled"? I mean: we are not hiding the alert, we are having device in normal operations state wrt to this check.

File: 01.Get-started/05.Monitor/docs.md [hunk 3]
```diff
@@ -171,23 +175,25 @@ Extract the container name:
 CONTAINER_NAME=$(docker ps | tail -1 | awk '{print $NF}')
 ```
 
-Create the file `dockerevents_testcontainer.sh` with the contents:
+Create the file `/etc/mender-monitor/monitor.d/available/dockerevents_testcontainer.sh` running the following command:
```
Comment (merlin-northern) [inline 01.Get-started/05.Monitor/docs.md:178]: so `mender-monitorctl` does not  support `create` subcommand on the `dockerevents` subsystem? if not then we need a ticket for it.

**Review by merlin-northern — COMMENTED** [review]
thanks. dropped some comments, treat as suggestions.

**Review by oleorhagen — APPROVED** [review]

**Comment by snusifer** [comment]
@merlin-northern 
> I would rephrase "the alert hides" somehow it does not seem a good fit, perhaps "the alert is cancelled"? I mean: we are not hiding the alert, we are having device in normal operations state wrt to this check.

Yeah, the alert is never hidden. What happens when the reader deletes the file is that the user will get an OK alert, with a green check mark. The old alert for when the user created the file will always stay in the alert history.

**Comment by snusifer** [comment]
With all the added `sudo`s now, i think reword [this](https://github.com/mendersoftware/mender-docs/blob/08629236c05d291da56d9a2bdb0796c55916bf22/01.Get-started/05.Monitor/docs.md?plain=1#L37-L39) a bit differently?

File: 01.Get-started/05.Monitor/docs.md (hunk 3 above)
Comment (tranchitella) [reply 01.Get-started/05.Monitor/docs.md:179]: Sorry, I don't get what you mean. Can you please follow-up on this topic?

File: 01.Get-started/05.Monitor/docs.md [hunk 4]
```diff
@@ -162,7 +167,7 @@ events and notifies you in the event that a container was restarted.
 First spin up the container to monitor:
 
 ```bash
-docker run --rm -d alpine sleep infinity
+sudo docker run --rm -d alpine sleep infinity
```
Comment (snusifer) [inline 01.Get-started/05.Monitor/docs.md:170]:
```suggestion
sudo docker run --name demo --rm -d alpine sleep infinity
```

File: 01.Get-started/05.Monitor/docs.md [hunk 5]
```diff
@@ -171,23 +176,25 @@ Extract the container name:
 CONTAINER_NAME=$(docker ps | tail -1 | awk '{print $NF}')
```
Comment (snusifer) [inline 01.Get-started/05.Monitor/docs.md:176]: We could just give this container a name, and not depend on that `$CONTAINER_NAME` variable. Then we can remove this step.

File: 01.Get-started/05.Monitor/docs.md (hunk 3 above)
Comment (merlin-northern) [reply 01.Get-started/05.Monitor/docs.md:179]: sure: https://tracker.mender.io/browse/MEN-5507

**Comment by tranchitella** [comment]
@snusifer @oleorhagen what's the status of this PR? Can we complete it and merge it?

**Comment by oleorhagen** [comment]
@snusifer has created a follow-up PR here: https://github.com/mendersoftware/mender-docs/pull/1713

This should be closed.
